smart_proxy_dynflow_core.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency('foreman-tasks-core', '>= 0.1.7')
   gem.add_runtime_dependency('sequel')
   gem.add_runtime_dependency('sqlite3')
-  gem.add_runtime_dependency('sinatra', '~> 1.4')
+  gem.add_runtime_dependency('sinatra')
```

Comment:
Should this be ```>= 1.4``` instead?

Reply:
I think we should be ok with whatever we get from the proxy, so I'm ok with this way

Reply:
The core part doesn't have to run with the proxy I think but I agree that chance is remote.